submission.json

```diff
@@ -1,6 +1,39 @@
 {
-  "id": 123,
-  "students": [],
-  "created_at": "2023-03-18T15:30:51.212+05:30",
-  "updated_at": "2023-03-18T15:30:51.212+05:30"
+  "id": 1496714,
+  "students": [110673],
```

Comment:
can we choose to keep just the keys that are required to test the action?. Also clean up the actual ids with test data.

Reply:
Done. Removed the unwanted files key.